core/oslc4j-core/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/ordfm/ResourcePackages.java

```diff
@@ -49,7 +49,7 @@ public class ResourcePackages {
     /**
      * The RDFs-Classes types mapping.
      */
-    static final Map<String, List<Class<?>>> TYPES_MAPPINGS = new HashMap<>();
+    static final Map<String, Set<Class<?>>> TYPES_MAPPINGS = new HashMap<>();
```

Comment:
@ben81 please note that the map is not `"org.eclipse.lyo.oslc4j.provider.jena.resources.child.ChildAnimal" -> [ChildAnimal.class]` but instead `"http://locahost:7001/vocabulary/AbstractTypesTest2"->[ChildAnimal.class]`. It can happen that you annotate multiple classes with the same `@OslcNamespace("http://locahost:7001/vocabulary/")`/`@OslcResourceShape(title = "AbstractTypesTest2")` annotation pair and a collection on the map value is designed to handle this.